openmc/opencg_compatible.py

```diff
@@ -398,7 +489,26 @@ def get_opencg_cell(openmc_cell):
 
 
 def get_compatible_opencg_cells(opencg_cell, opencg_surface, halfspace):
-
+    """Generate OpenCG cells that are compatible with OpenMC equivalent to an OpenCG
+    cell that is not compatible.
+
+    Parameters
+    ----------
+    opencg_cell : opencg.Cell
+        OpenCG cell
+    opencg_surface : opencg.Surface
+        OpenCG surface that causes the incompatibility, e.g. an x-squareprism
```

Comment:
Might change "x-squareprism" to "XSquarePrism." The former might confuse some users since "x-squareprism" closely parallels OpenMC's surface type strings "x-plane," "y-plane," etc. while the latter more explicitly refers to the OpenCG class name.